ndohyep/templates/comments/report_response.html

```diff
@@ -0,0 +1,17 @@
+{% extends "core/_article_page.html" %}
+{% load wagtailcore_tags wagtailimages_tags comments mptt_tags molo_commenting_tags i18n %}
+
+{% block content %}
+<div class="block {{article.get_parent_section.get_effective_extra_style_hints}}">
+  <br>
+  <h4 class="left">{% trans "Thanks!" %}</h4>
+</div>
+<div class="block">
+  <br>
+  <p><strong>{% trans "This comment has been reported." %}</strong>
+  {% trans "It will be reviewed by the B-Wise team and may be removed if it breaks our Platform rules" %}</p>
+</div>
+<div class="title {{article.get_parent_section.get_effective_extra_style_hints}}">
+  <a href="javascript:history.back();" class="button">{% trans "Back" %}</a>
```

Comment:
This javascript will likely break in many situations. Can we not just pass along the link (or even article-id) of the original article and create a regular `href` link?